ROADMAP.md

```diff
@@ -44,4 +33,4 @@ with greater flexibility:
 - `LocalsSlot`
 - `StaticsSlot`.
 
-Removal of the `Dump*Slot()` methods are scheduled for the 0.108.0 release.
\ No newline at end of file
+Removal of the `Dump*Slot()` methods are scheduled for the 0.108.0 release.
```

Comment:
Not related to this commit.

Reply:
Yeah, but we have a broken text file. I've postponed this change to the last commit and can make it a separate one if you wish.